### PR TITLE
[RPS-433] Category facet will be displayed in a tree

### PR DESCRIPTION
**Pull request by jacsamell**
Group the taxonomy facets into the taxonomy tree structure.
Also bumped up the application version as we have gone live
again on beta!

### CODE REVIEW

File: acceptance-tests/src/test/resources/features/site/facets.feature [hunk 1]
```diff
@@ -129,21 +116,61 @@ Feature: Faceted search
...
 
     Scenario: Upcoming publications are returned
         Given I navigate to the "search" page
-        And I click on the accept cookie banner
         Then I should see the list with title "PUBLICATION STATUS" containing:
             | Published ( ... |
             | Upcoming (2) |
         When I click on the "Upcoming" button
         Then I can click on the "Upcoming Publication" link
+@WIP
```
Comment (ngta1) [inline acceptance-tests/src/test/resources/features/site/facets.feature]: Remove WIP :-)

File: site/src/main/java/uk/nhs/digital/common/components/TaxonomyFacetWrapper.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        }
+
+        // link the taxonomy facets to the taxonomy category
+        HashMap<String, TaxonomyFacet> keyToTaxonomyFacet = new HashMap<>(taxonomyFacets.size());
+        for (HippoFolderBean facet : taxonomyFacets) {
+            String key = facet.getName();
+            keyToTaxonomyFacet.put(key, new TaxonomyFacet(taxonomy, key, facet));
+        }
+
+        // build the tree structure
```
Comment (ngta1) [inline site/src/main/java/uk/nhs/digital/common/components/TaxonomyFacetWrapper.java]: 👍

File: acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/SitePage.java [hunk 3]
```diff
@@ -94,8 +94,12 @@ public WebElement findPageElement(String elementName, int nth) {
         return null;
     }
 
-    public WebElement findPageElementById(String elementId) {
-        return helper.findElement(By.id(elementId));
+    public void clickCookieAcceptButton() {
```
Comment (ngta1) [inline acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/SitePage.java]: 👍saves a few lines on the feature files

File: ci-cd/Makefile [hunk 4]
```diff
@@ -12,7 +12,7 @@ RELEASE_TIME_UTC = $(shell ((noon_utc=12-$$(date +%-H)+$$(TZ=UTC-24 date +%-H)))
 TRAVIS_BUILD_CMD ?= test.site
 TRAVIS_EVENT_TYPE ?=
 VENV ?= $(PWD)/.venv
-VERSION ?= $(shell git describe --tags --match "$(VERSION_BASE)*")
+VERSION ?= $(shell git describe --tags --match “$(VERSION_BASE)*” || echo $(VERSION_BASE).0)
```
Comment (ngta1) [inline ci-cd/Makefile]: Assume change to curved double quotes runs fine?

Comment (jacsamell) [reply ci-cd/Makefile]: No this would have probably broken, thanks!

**Pull request merged**